extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -13,3 +13,45 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+/* DRAFT!!
+*/
+
+amp-accordion > section > :first-child {
+  cursor: pointer;
+  background-color: #efefef;
+  padding-right: 20px;
+  border: solid 1px #dfdfdf;
+}
+
+/* heading/clontent */
+.i-amphtml-accordion-header,
+.i-amphtml-accordion-content {
+  margin: 0;
+}
+
+/* heading
+ * TODO(dvoytenko): remove most of these styles, except maybe for `cursor`.
+ */
+.i-amphtml-accordion-header {
+  cursor: pointer;
+  background-color: #efefef;
+  padding-right: 20px;
+  border: solid 1px #dfdfdf;
+}
+
+amp-accordion.i-amphtml-display-locking > section:not([expanded]) > :last-child,
```

Comment:
All separate PR for display locking. But for now:

1. _I think_ the `visibility:hidden` was related to this.
2. I think this should definitely only be on Preact level. It may or may not need to be in a stylesheet. Inline styles might be better for this.

Reply:
Updated to inline styles on Preact side.  Have removed `visibility:hidden` for now.  Can look into it with display locking.